x-pack/plugins/observability/public/pages/alert_details/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
@benakansara I propose not to include implementation in `index` files and only use them for exports.
Similar to what we did in this refactoring: https://github.com/elastic/kibana/pull/119211/files